std-bits/table/src/main/java/org/enso/table/util/NameDeduplicator.java

```diff
@@ -36,11 +36,11 @@ public String makeValid(String input) {
     return input;
   }
 
-  public List<String> makeUnique(List<String> names) {
+  public List<String> makeUniqueList(List<String> names) {
     return names.stream().map(this::makeUnique).collect(Collectors.toList());
   }
 
-  public String[] makeUnique(String[] names) {
+  public String[] makeUniqueVector(String[] names) {
```

Comment:
```suggestion
  public String[] makeUniqueArray(String[] names) {
```

It is returning a Java array, not an Enso vector - IMO the name is confusing as it is now.